releasing_guide.md

```diff
@@ -18,9 +18,13 @@ To publish a new LumiSpy release do the following steps:
 - Create a tag e.g. `git tag -a v0.1.1 -m "LumiSpy version 0.1.1"`
 - Push tag to user fork for a test run `git push origin v0.1.1`. Will run the release
   workflow without uploading to PyPi
-- Push tag to LumiSpy repository to trigger release `git push upstream v0.1.1`
-  (this triggers the GitHub action to create the sdist and wheel and upload to
-  PyPi automatically). :warning: this is a point of no return :warning:
+- Push tag to LumiSpy repository to trigger release `git push upstream v0.1.1`. 
+  :warning: this is a point of no return :warning:
+  - triggers the GitHub action to create the sdist and wheel and upload them to
```

Comment:
Currently, the order is different:

1. create github release, which trigger the zenodo release
2. build wheel/sdist
3. upload to pypi

Reply:
You set it up in a different order for RSciIO - is there a particular advantage for that?

Reply:
Yes, see [comment below ](https://github.com/LumiSpy/lumispy/pull/191#discussion_r1208392335) and additional information section in https://github.com/hyperspy/rosettasciio/pull/126/files#diff-1ac981832aad6ef0ee37d78805ad59ce5cbd618646f64e363246b89acd57b312